### PR TITLE
Task3

### PR DESCRIPTION
**Pull request by LenaEvgena**
Task: https://github.com/Front-Grodno/Front-Grodno-JS-course-2021/blob/master/modules/react/react-3-components.md
main
Deploy: -
Deadline: -

+Implement HTML markup for required design for Home page (Only UI part) with all necessary components and apply styles for it.
+Use <ErrorBoundary> component for catching and displaying errors. You could create one component and wrap all your application or use several components.
+The task can be done with mocked data.
+Implement markup and styles for Add movie, Edit, Delete modal windows and Sorting on Home page.

### CODE REVIEW

File: src/components/App/App.jsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import Header from '../header/Header';
+import Footer from '../footer/Footer';
+import Main from '../main/main';
+import ErrorBoundary from '../errorBoundary/ErrorBoundary';
+import './style.scss';
+
+const App = () => {
+  return (
+    <>
```
Comment (SergeiKSS) [inline src/components/App/App.jsx:10]: Думаю в данном случае return можно не писать
const App = () => (
<> 
разметка ... 
</>
);

File: src/components/addButton/AddButton.jsx [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import './addButton.scss';
+
+const AddButton = (props) => {
+  return (
```
Comment (SergeiKSS) [inline src/components/addButton/AddButton.jsx:5]: const AddButton = (props) => (
<div> ... </div> );

File: src/components/deleteModal/DeleteModal.jsx [hunk 3]
```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import Footer from '../footer/Footer';
+import LogoTitle from '../logoTitle/LogoTitle';
+import SubmitButton from '../submitButton/SubmitButton';
+import './deleteModal.scss';
+
+const DeleteModal = (props) => {
+  return (
+    <div className="delete__modal">
```
Comment (SergeiKSS) [inline src/components/deleteModal/DeleteModal.jsx:9]: const DeleteModal = (props) => (
<div> ... </div>
);

File: src/components/footer/Footer.jsx [hunk 4]
```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import LogoTitle from '../logoTitle/LogoTitle';
+import './footer.scss';
+
+const Footer = () => {
+  return (
+    <footer className="footer">
```
Comment (SergeiKSS) [inline src/components/footer/Footer.jsx:7]: const Footer = () => (
<footer> ...

File: src/components/App/App.jsx (hunk 1 above)
Comment (LenaEvgena) [reply src/components/App/App.jsx]: Done

File: src/components/addButton/AddButton.jsx (hunk 2 above)
Comment (LenaEvgena) [reply src/components/addButton/AddButton.jsx]: Done

File: src/components/deleteModal/DeleteModal.jsx (hunk 3 above)
Comment (LenaEvgena) [reply src/components/deleteModal/DeleteModal.jsx:18]: Done

File: src/components/header/Header.jsx [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+    super(props);
+    this.state = {
+      showMovieModal: false,
+    };
+    this.handleMovieModal = this.handleMovieModal.bind(this);
+  }
+
+  handleMovieModal() {
+    this.setState({ showMovieModal: !this.state.showMovieModal });
+  }
```
[the remark this answers is not in the document]
Comment (LenaEvgena) [reply src/components/header/Header.jsx:23]: Done

File: src/components/footer/Footer.jsx (hunk 4 above)
Comment (LenaEvgena) [reply src/components/footer/Footer.jsx]: Done

**Review by SergeiKSS — APPROVED** [review]

**Pull request merged**